src/Control/Effect/NonDet.hs

```diff
@@ -10,13 +10,13 @@ module Control.Effect.NonDet
 , optional
 , MonadPlus(..)
   -- * Constituent effects
-, Control.Effect.Choose.Choose (..)
-, Control.Effect.Empty.Empty (..)
+, module Control.Effect.Choose
+, module Control.Effect.Empty
 ) where
 
 import Control.Applicative (Alternative(..), optional)
-import Control.Effect.Choose hiding ((<|>), many, some, optional)
-import Control.Effect.Empty hiding (empty, guard)
+import Control.Effect.Choose (Choose(..))
+import Control.Effect.Empty (Empty(..))
```

Comment:
Been meaning to do this for a while—it’s simpler to only list what we want to re-export.